lib/widget/youtube/native/youtube_platform_native.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:ensemble/widget/youtube/youtubestate.dart';
+
+YoutubeWeb getInstance() => YoutubeNativeInstance();
+
+class YoutubeNativeInstance implements YoutubeWeb {
```

Comment:
doesn't make sense for YoutubeNativeInstance to implement YoutubeWeb. Your code should be understandable and nicely written. This doesn't make sense